django/db/backends/mysql/features.py

```diff
@@ -46,7 +46,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             SET V_I = P_I;
         END;
     """
-    supported_explain_formats = {'JSON', 'TRADITIONAL'}
+    # MySQL has TRADITIONAL instead of TEXT, but we alias TRADITIONAl to TEXT for consistency
```

Comment:
`TRADITIONAl` → `TRADITIONAL` (You let go of your shift key too early!)

Perhaps reword to `# Add TEXT as an alias of TRADITIONAL for consistency with other backends.`